tests/Cache/FileCacheTest.php

```diff
@@ -74,14 +71,7 @@ public function testBrokenCacheRead(): void
             'broken_feature',
             new FeatureNode(null, null, [], null, [], '', '', null, 1),
         );
-        $files = iterator_to_array(
-            (new Finder())
-                ->in($this->path)
-                ->depth(1)
-                ->name('*.feature.cache')
-                ->files(),
-            false
-        );
+        $files = Filesystem::find("$this->path/**/*.feature.cache");
```

Comment:
this `**` in the glob does not match the `->depth(1)` config of the Finder.

Reply:
True - but arguably for this test it is safer to assert that the previous `->write` has only created one file at any depth. Ultimately we're trying to make sure that the file we modify below is the one the cache is expecting to read.

Reply:
I'm not sure why we're even discussing this since I introduced the finder implementation in one commit (instead of glob with **) and then changed it back (to **) in a subsequent one. 🤔